scanner/scanner_test.go

```diff
@@ -1,26 +1,27 @@
-package workers
+package scanner_test
```

Comment:
No functional changes.

The only thing changed in this file is the new test package name which is used in order to work around a cyclical dependency with the test package.